src/rest_api.py

```diff
@@ -44,6 +53,55 @@ def liveness():
     return flask.jsonify({}), 200
 
 
+@app.route('/api/v1/sync-graph-data', methods=['POST'])
+@login_required
+def sync_data():
+    """
+    Endpoint for carrying a sync of graph db data.
+
+    Takes in value to either call sync of non cve version
+    or latest version or both.
+
+    valid input: {
+        "non_cve_sync": True / False,
+        "latest_version_sync": True / false,
```

Comment:
@yzainee  typo. should have been `true / false`.

Reply:
changed